app/seksjon/barnetillegg/v1/komponenter/BarnModal.tsx

```diff
@@ -79,6 +80,7 @@ export function BarnModal({ ref, spørsmålId }: IProps) {
 
     const nyttDokumentkrav: Dokumentasjonskrav = {
       id: dokumentasjonskravId,
+      seksjonId: seksjonId,
```

Comment:
Har behov for å legge til `seksjonId`. Vi trenger det for 
`const url =´${getEnv("DP_SOKNAD_ORKESTRATOR_URL")}/soknad/${soknadId}/${seksjonId}/dokumentasjonskrav´;`